.github/workflows/docs.yml

```diff
@@ -0,0 +1,34 @@
+name: Test Docs Build
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ./.github/actions/prepare
+        with:
+          node-version: 22
```

Comment:
Why not use the `node-version-file` file here? Maybe the `prepare` action could just default to using `.nvmrc`.